utils/dataset_manifest/create.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (C) 2021 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+import argparse
+import os
+import sys
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--type', type=str, choices=('video', 'images'), help='Type of datset data', required=True)
+    parser.add_argument('manifest_directory',type=str, help='Directory where the manifest file will be saved')
+    parser.add_argument('--chunk_size', type=int,
+        help='Chunk size that will be specified when creating the task with specified video and generated manifest file')
+    parser.add_argument('sources', nargs='+', help='Source paths')
```

Comment:
It will be ready difficult to create a manifest file for a huge dataset with the approach. The command line has some length limitations (I don't remember exactly but probably you cannot have it longer than 8192 chars). It is better to have ability to provide a directory or a file. Usually you will have a video file or a directory with images.